lib/Mail/DMARC/Report/Sender.pm

```diff
@@ -48,13 +48,14 @@ sub set_transports_method {
 # Return a list of transports to try in order.
 sub get_transports_for {
     my ( $self, $args ) = @_;
-
     # Have we passed a custom transports generation class?
     if ( $self->{transports_method} ) {
-        return &{$self->{transports_method}}( $args );
+        my @transports = &{$self->{transports_method}}( $args );
+        return @transports;
```

Comment:
Is this to coerce the returned type?

Reply:
No, It was purely to aid debugging.
For info, I'm waiting for a change to Email::Sender::Transport::SMTP to be merged before merging this change.